Automattic/jetpack
# Social: Move Redux store to publicize-components

This change moves the store over from Social to the publicize-components package.

## Proposed changes:

* Moved the store over into `social-store` in 80a1fde04c6a42035e06acb4817f556f413960ad
* Updated the imports and all the usage to use the store from publicize-components in 736029122ffbd1c1da49c1e2418d68cc1048620f
* Removed the old store in acd2b38cf5795db40f2ff5085c2be5e4aa16e454

### Other information:


## Jetpack product discussion


1205142973434174-as-1205212271186925
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Review the changes, it is easier to do by following the commits.
* Check that Jetpack Social works with the plugin enabled, and with it disabled and Jetpack enabled
  * SIG settings should be saved correctly on changes
  * Publishing to social media should work
  * Check on Simple and Atomic sites that it's working too
